telemetry/definitions/commonDefinitions.json

```diff
@@ -2663,6 +2704,89 @@
             "description": "User has accepted the approach generated by Weaverbird.",
             "metadata": [{ "type": "enabled" }, { "type": "awsqConversationId" }] 
         },
+        {
```

Comment:
I see that a few things are added but none are deleted, are we going to remove all the awsq ones that are currently being used here or is that a next step?

Reply:
I think to make it backwards compatible.
I think it is better to keep awsq_ metrics until we are fully moved to aamazonq_
what do you think>

Reply:
I think that's fine so we don't block other teams but we have to have a task to make sure to remove them

Reply:
I will add one

Reply:
Is the change live?

Reply:
no, I this is to register new events.I will open second CR in aws-toolkit-vscode-staging

Reply:
Yes, we've had awsq metrics for ~1.5 weeks